data/README.md

```diff
@@ -0,0 +1,24 @@
+# Data Directory Organization
```

Comment:
I think we need some best practices here. For example, GitHub doesn't love large files and recommends files stay under 50 MB and has a hard limit of 100 MB. GH also recommends repos stay under 1 GB.

So, we should specify that these are **minimal** examples. Maybe aim for 100-1000 lines in any data file, more if absolutely required to reproduce an issue with the spec.

Reply:
Love this. Will add.